examples/docker/docker-compose-minimal.yml

```
@@ -0,0 +1,24 @@
+# This docker-compose file starts the required services for wasmCloud, which is just:
+#   a NATS server
+#   a wasmCloud host
+
+version: "3"
+services:
+  nats:
+    image: nats:2.10.7-alpine
```

So that we don't need to keep up with all of the new releases of NATS, maybe we pin to the 2.10.x branch instead:
```suggestion
    image: nats:2.10-alpine
```

---

I know this is different than what we do in the other files, I think we should go back and update the other files to use the same version.

---

I'm okay with this change, but think it can be done in a sweeping chore. In fact, happy to do that now

---

https://github.com/wasmCloud/wasmCloud/pull/1446